buildSrc/src/main/kotlin/CrateSet.kt

```diff
@@ -3,45 +3,57 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+data class Crate(val name: String, val versionPropertyName: String)
+
 object CrateSet {
+    const val STABLE_VERSION_PROP_NAME = "smithy.rs.runtime.crate.stable.version"
+    private const val UNSTABLE_VERSION_PROP_NAME = "smithy.rs.runtime.crate.unstable.version"
```

Comment:
> Does some part of the release process rely solely on the stable version?

Yes it does, except until I have added the unit in 4add629. After that commit, both of these variables are no longer private since they are used by the test.